.gitignore

```diff
@@ -1,2 +1,3 @@
-/composer.lock
-/vendor/
```

Comment:
I don't think these slashes are redundant; Leading slashes scope the match to the current directory, which is the intention here. Trailing slashes limit the match to directories, which is the intention for `/vendor/`.

Absence of a leading slash should be used for files expected to pop up throughout the directory hierarchy. I'd rather start specific than risk future files being accidentally ignored.

You could add `/phpunit.xml` as a separate PR if you want.